content/en/docs/concepts/overview/components.md

```diff
@@ -125,7 +125,11 @@ saving container logs to a central log store with search/browsing interface.
 
 ## {{% heading "whatsnext" %}}
 
-* Learn about [Nodes](/docs/concepts/architecture/nodes/)
-* Learn about [Controllers](/docs/concepts/architecture/controller/)
-* Learn about [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/)
-* Read etcd's official [documentation](https://etcd.io/docs/)
+Learn more about the following:
+   * [Nodes](/docs/concepts/architecture/nodes/) and communication between [Nodes and control plane](/docs/concepts/architecture/control-plane-node-communication/).
```

Comment:
```suggestion
   * [Nodes](/docs/concepts/architecture/nodes/) and [communication between Nodes and control plane](/docs/concepts/architecture/control-plane-node-communication/).
  
```
Why not select "communication between Nodes and control plane]" in the square brackets as the link? The title that this link points to is "Communication between Nodes and the Control Plane"